docker-compose.hobby.yml

```
@@ -287,10 +288,8 @@ services:
             service: livestream
         environment:
             - JWT.TOKEN=${POSTHOG_SECRET}
```

```suggestion
            - LIVESTREAM_JWT_SECRET=${POSTHOG_SECRET}
```
this should do the trick